sentinel-core/src/main/java/com/alibaba/csp/sentinel/Entry.java

```diff
@@ -52,14 +54,19 @@
 
     private static final Object[] OBJECTS0 = new Object[0];
 
-    private long createTime;
+    private final long createTime;
+    private long completeTimestamp;
```

Comment:
How about keeping the same suffix of the two fields?

Reply:
Good idea. I've updated `createTime` to `createTimestamp`.